docs/forensics/b07_context_inventory_evidence.md

```
@@ -0,0 +1,399 @@
+
+### B1 — Apply migrations on a clean DB (blocked)
+
+Attempt to start Postgres via compose:
+
+```powershell
+docker compose -f docker-compose.e2e.yml up -d postgres
+```
+
+Observed output (daemon unavailable):
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)